packages/extension-ui/src/components/EditMenuCard.tsx

```diff
@@ -57,44 +58,59 @@ const ExtraContent = ({ extra = 'chevron' }: ExtraProps): React.ReactElement<Ext
   }
 };
 
+const StyledLink = styled.a`
+  text-decoration: none;
+  color: inherit;
+`;
+
 function EditMenuCard({
   className,
   description,
   extra,
   isDanger = false,
+  link,
   onClick,
   preIcon,
   title,
   toggle
 }: Props): React.ReactElement<Props> {
   return (
-    <div
-      className={className}
-      onClick={onClick}
+    <StyledLink
+      href={link}
+      rel='noreferrer'
+      target='_blank'
     >
-      <div className='flex-container'>
-        <div className='flex-group'>
-          <div className={`title ${isDanger ? 'danger' : ''}`}>
-            {preIcon}
-            {title}
-          </div>
-          <div className='description'>
-            <span className='description-text'>{description}</span>
-            {toggle && <div className='extra'>{toggle}</div>}
-            {!toggle && (
-              <div className='extra'>
-                <ExtraContent extra={extra} />
-              </div>
-            )}
+      <div
+        className={className}
+        onClick={onClick}
+      >
+        <div className='flex-container'>
+          <div className='flex-group'>
+            <div className={`title ${isDanger ? 'danger' : ''}`}>
+              {preIcon}
+              {title}
+            </div>
+            <div className='description'>
+              <span className='description-text'>{description}</span>
+              {toggle && <div className='extra'>{toggle}</div>}
+              {!toggle && (
+                <div className='extra'>
+                  <ExtraContent
+                    extra={extra}
+                    link={link}
+                  />
+                </div>
+              )}
```

Comment:
In both cases you use class extra so this check could be like this or smth similar
```jsx
 <div className='extra'>
  toggle ? {toggle} :  <ExtraContent
       extra={extra}
        link={link}
     />
</div>
```